src/hotspot/share/gc/g1/g1CardSet.hpp

```diff
@@ -47,6 +47,7 @@ enum G1AddCardResult {
 };
 
 class G1CardSetConfiguration {
+  // #bits required to hold the max card index; log(#cards_per_region)
```

Comment:
I would prefer a real sentence like `// [Holds] the number of bits required to cover the maximum card index for the regions covered by this card set.`.

Reply:
Fixed.